content/blog/building-an-alternative-service-mesh.md

```diff
@@ -35,7 +35,7 @@ One class of software implements the SMI directly by consuming the APIs defined
 While these tools may not directly implement the functionality covered by the SMI, they play an essential role in the ecosystem by adopting and managing the technologies which support the SMI specification:
 
 *   Weaveworks Flagger – functions as a control plane for Kubernetes deployment while supporting multiple service mesh implementations
-*   Service Mesh Hub – serves as a management plane for several Service Mesh implementations
+*   Meshery – serves as a management plane for several Service Mesh implementations
```

Comment:
I think these are actually two things? Not a rename of one.

Reply:
@bridgetkromhout you're right in that these were two different things. Considering that Service Mesh Hub isn't a thing anymore and that an entry for Meshery is missing, I figured that the same entry could be reused. I'll update to add a new entry, leaving the stale entry for someone else to delete.

Reply:
Right, Gloo Mesh is their new name, but I don't think it makes sense to ret-con it in an old blog post.

Reply:
Sounds real good. Yes, understanding that it is a blog post now... this makes much sense. Updating...

Reply:
Updated. 👍